src/components/sidebar/sidebar.js

```diff
@@ -24,23 +31,23 @@ export const BSidebar = /*#__PURE__*/ Vue.extend({
     },
     right: {
       type: Boolean,
-      default: false
+      default: () => getComponentConfig(NAME, 'right')

```

Comment:
We don't allow boolean only props to be in the config.

So this should be hard coded to `false`

Reply:
fixed